ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -153,7 +153,7 @@ export default function CollectiblesItems({
     <div className="collectibles-items">
       <Box padding={[6, 4]} flexDirection={FLEX_DIRECTION.COLUMN}>
         <>
-          {Object.keys(collections).map((key) => {
+          {collectionsKeys.map((key) => {
```

Comment:
Question about this and a suggestion for a future PR... Do collections need to be displayed in a certain order? If that's the case, then we may not want to use an object to store those collections, because the key/value pairs that are stored in objects are inherently unordered. It seems like it might be valuable to use an array to keep them, or at least, have a separate array that tracks the correct order of the keys.

Reply:
I don't know of any reason that the collections ought to be displayed in any order yet. In the future there could and maybe should be a way for users to order/organize them as they like.